### PR TITLE
chore(auth): resolve swiftformat errors and warnings

### PR DESCRIPTION
**Pull request by phantumcode**
## Description
Resolving SwiftFormat errors and warnings for the Auth plugin.

## General Checklist

- [ ] Added new tests to cover change, if needed
- [ ] Build succeeds with all target using Swift Package Manager
- [ ] All unit tests pass
- [ ] All integration tests pass
- [ ] Security oriented best practices and standards are followed (e.g. using input sanitization, principle of least privilege, etc)
- [ ] Documentation update for the change if required
- [ ] PR title conforms to conventional commit style
- [ ] New or updated tests include `Given When Then` inline code documentation and are named accordingly `testThing_condition_expectation()`
- [ ] If breaking change, documentation/changelog update with migration instructions

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/ResolverTests/AuthorizationState/FetchAuthSessionStateResolverTests.swift [hunk 1]
```diff
@@ -30,7 +31,7 @@ class FetchAuthSessionStateResolverTests: XCTestCase {
 //        let cognitoSession = AWSAuthCognitoSession.testData
 
         let validSequences: [FetchAuthSessionStateSequence] = [
-//            StateSequence(oldState: .initializingFetchAuthSession,
+            //            StateSequence(oldState: .initializingFetchAuthSession,
```
Comment (thisisabhash) [inline AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/ResolverTests/AuthorizationState/FetchAuthSessionStateResolverTests.swift:34]: seems like an unintentional change?

File: AmplifyPlugins/Auth/Tests/AuthHostApp/AuthIntegrationTests/Helpers/TOTPHelper.swift [hunk 2]
```diff
@@ -43,22 +44,22 @@ struct TOTPHelper {
...
-        15,16,17,18, 19,20,21,22, 23,24,25,__, __,__,__,__,  // 0x70 - 0x7F
-        __,__,__,__, __,__,__,__, __,__,__,__, __,__,__,__,  // 0x80 - 0x8F
-        __,__,__,__, __,__,__,__, __,__,__,__, __,__,__,__,  // 0x90 - 0x9F
-        __,__,__,__, __,__,__,__, __,__,__,__, __,__,__,__,  // 0xA0 - 0xAF
-        __,__,__,__, __,__,__,__, __,__,__,__, __,__,__,__,  // 0xB0 - 0xBF
-        __,__,__,__, __,__,__,__, __,__,__,__, __,__,__,__,  // 0xC0 - 0xCF
-        __,__,__,__, __,__,__,__, __,__,__,__, __,__,__,__,  // 0xD0 - 0xDF
-        __,__,__,__, __,__,__,__, __,__,__,__, __,__,__,__,  // 0xE0 - 0xEF
-        __,__,__,__, __,__,__,__, __,__,__,__, __,__,__,__,  // 0xF0 - 0xFF
+        __, __, __, __, __, __, __, __, __, __, __, __, __, __, __, __,  // 0x00 - 0x0F
```
Comment (thisisabhash) [inline AmplifyPlugins/Auth/Tests/AuthHostApp/AuthIntegrationTests/Helpers/TOTPHelper.swift:47]: not sure about the rule being enforced here, should we keep the old changes as comments are aligned?

File: AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/ResolverTests/AuthorizationState/FetchAuthSessionStateResolverTests.swift (hunk 1 above)
Comment (phantumcode) [reply AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/ResolverTests/AuthorizationState/FetchAuthSessionStateResolverTests.swift]: looks to be attempting to resolve alignment, I've removed the commented out code.

File: AmplifyPlugins/Auth/Tests/AuthHostApp/AuthIntegrationTests/Helpers/TOTPHelper.swift (hunk 2 above)
Comment (phantumcode) [reply AmplifyPlugins/Auth/Tests/AuthHostApp/AuthIntegrationTests/Helpers/TOTPHelper.swift:47]: space after commas are being enforced here, I've updated the comments to be aligned as well.

**Review by thisisabhash — APPROVED** [review]

**Pull request merged**